canonical-websites/cn.ubuntu.com
# Use talisker in production Dockerfile

QA
--

``` bash
$ docker build --build-arg COMMIT_ID=`git rev-parse HEAD` --tag cn .
$ docker run -p 80:80 cn
...
2018-03-22 16:23:14.161Z INFO gunicorn.error "Starting gunicorn 19.7.1"
2018-03-22 16:23:14.162Z INFO gunicorn.error "Listening at: http://0.0.0.0:80 (1)"
...
```

See talisker output ^. Go to http://localhost:8010 and http://localhost:8010/_status, check they work!
